<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountWidget</class>
 <widget class="QWidget" name="AccountWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>148</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="label_UserIcon">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_DisplayName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>DisplayName</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_EmailAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>EmailAddress</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="buddy">
      <cstring></cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
